llvm/lib/CodeGen/RegAllocFast.cpp

```diff
@@ -332,10 +334,11 @@ class RegAllocFastImpl {
 private:
   void allocateBasicBlock(MachineBasicBlock &MBB);
 
-  void addRegClassDefCounts(MutableArrayRef<unsigned> RegClassDefCounts,
-                            Register Reg) const;
+  void addRegClassCounts(MutableArrayRef<unsigned> RegClassDefCounts,
```

Comment:
Parameter name RegClassDefCounts is now out of sync

Reply:
Done